akka-actor/build.sbt

```diff
@@ -5,5 +5,9 @@ Formatting.formatSettings
 OSGi.actor
 Dependencies.actor
 Version.versionSettings
+unmanagedSourceDirectories in Compile += {
+  val base = scalaVersion.value.take(4)
+  sourceDirectory.value / s"main-$base" / "scala"
+}
```

Comment:
That is only there automatically if `crossPaths` is set to `true` which it isn't in akka.